README.md

```diff
@@ -1,9 +1,31 @@
 # Northcoders News API
+--Setting Up the Environment--
+-Environment Variables-
+You will need to create two .env files to connect the project to the correct PostgreSQL databases for development and testing:
+1-.env.development
```

Comment:
To make it clear which part is the filename, you could wrap that part in backticks: `.env.development`